server/jetstream.go

```diff
@@ -91,6 +91,7 @@ type JetStreamAccountStats struct {
 }
 
 type JetStreamAPIStats struct {
+	Level    string `json:"level"`
```

Comment:
should we maybe make it an int? we need it string on metadata cos thats all metadata uses - but its better as a int here for callers as that clearly indicates the format of this thing where `"1"` makes someone wonder if `"1.a"` is also valid etc?

Reply:
Yeah, think that would simplify API level a lot. Currently it's just an int in the form of a string, but would technically allow for full on semver and "1.a".

For comparisons int is definitely easier, where the API level would just be upped if we introduce a new feature that needs it.

Should I adapt these PRs to use int?

Reply:
Yeah I think so, api level will always just +1 no semver etc - we have to assume we'll introduce new features mid release cycle, so we need to be able to up this thing in non semver ways

Reply:
Done